samples/UserPassOrTokenClient.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+"""
+Get a users room list and indicate login type.
+
+to use user+pass to login and get a token:
+    arg usage: --host 'host' --user 'username' --password 'password'
+to use user+token to login:
+    arg usage: --host 'host' --user 'username' --token 'token'
+"""
+
+import argparse
+from matrix_client.client import MatrixClient
+
+
+def _example(host, user, password, token):
```

Comment:
Could I trouble you to use ``example`` over ``_example``. It's a small thing, but I just find for samples it tends to be easier to keep names as simple as they can be.